File: client/src/Routines/Routine.js
import React from "react";
import { Link } from "react-router-dom";

//In charge of each Routine
export const Routine = ({ routine }) => {
    console.log(routine)
  return (
    
      <tbody>
        <tr>
          <td> {routine.intention}</td>
          <td>{`${routine?.users?.displayName}`}</td>
          <td>{`${routine.cycles}`}</td>
          <td>{`${routine?.poses?.name}`}</td>
          <td>{`${routine.reflection}`}</td>
          <td><Link to={`/routines/${routine.id}`}>Details</Link></td>
        </tr>
      </tbody>
  );
};

File: SoulFly/SoulFly/Models/Routine.cs
using System.ComponentModel.DataAnnotations;

namespace SoulFly.Models
{
    public class Routine
    {
        public int? Id { get; set; }
        public int? UserId { get; set; }
       
        public string? Intention { get; set; }
        public int? PoseId { get; set; }
        public int? Cycles { get; set; }
        public DateTime? CreationDate { get; set; }
        public string? Reflection { get; set; }
        public Poses? Poses { get; set; }
        public Users? Users { get; set; }


       
    }
}
